sdks/python/apache_beam/testing/analyzers/perf_analysis_utils.py

```diff
@@ -170,6 +170,10 @@ def find_latest_change_point_index(metric_values: List[Union[float, int]]):
   if not change_points_indices:
     return None
   change_points_indices.sort()
+  # If the latest change point is the last index of the metric_values,
+  # then consider it as no change point since it might be a flake.
+  if len(metric_values) - 1 == change_points_indices[-1]:
```

Comment:
I checked a similar example, 

```
data = [1] * 100 + [50]  -> alert
data = [1] * 100 + [50] + [1] -> alert
data = data = [1] * 100 + [50] + 2 * [1] -> no alert
```

I think this should be fine.

Reply:
data = [1] * 100 + [50]  -> alert
data = [1] * 100 + [50] + [1] -> alert

why do these result in an alert? this looks like an anomaly in both cases.

Reply:
Since I think the difference in values are very high, edivisive considers the above examples as alerts. But after seeing couple more data points, edivisive deduces that it is a flake. 

>> why do these result in an alert? this looks like an anomaly in both cases.
For us, anomaly is alert right?

Reply:
We don't want to receive alerts for flake; i think we should wait for more runs until e-divisive can make a determination whether it's a flake or not. The reason for choosing e-divisive in the beginning was that it would help prevent unactionable alerts.

Reply:
Okay. this could be an edge case where the edivisive, if there is a massive spike in the recent metric data is outputting an alert. We can put a limit on the number of days we could avoid as change point. I will go with 3 days and if there is change point that was in the recent 3 days, we could avoid it. How does it sound?

Reply:
i think we are talking about  # of runs, not days.

> this could be an edge case where the edivisive

Do we understand why this is happening if the spike is specifically on last run but not other runs ?

Overall, sg if this workaround works.

Reply:
Yes, number of runs. 
>> Do we understand why this is happening if the spike is specifically on last run but not other runs ?
I understand why it is not an alert when the spike is not the last index but it should be the same case for the last index as well but the tool is sending an alert. This could be an edge case maybe.

Reply:
we should understand edivisive's behavior to use it appropriately.

Reply:
FYI, good read on edivisive - https://dl.acm.org/doi/abs/10.1145/3358960.3375791 explains how edivisive can be used(similar to how we use).